thirdparty/README.md

```diff
@@ -665,6 +665,17 @@ They can be reapplied using the patches included in the `vhacd`
 folder.
 
 
+## volk
+
+- Upstream: https://github.com/zeux/volk
+- Version: 1.2.170.0 (b18f3192886981f28d17f8f41c26c5a53ac3b977, 2021)
+- License: MIT
+
+Files extracted from upstream source:
+
+- `volk.h`, `volk.c`
+- `LICENSE.md`
+
 ## vulkan
 
 - Upstream: https://github.com/KhronosGroup/Vulkan-Loader
```

Comment:
We should probably change upstream and track https://github.com/KhronosGroup/Vulkan-Headers directly.

Reply:
Fixed